packages/ckeditor5-select-all/src/selectallcommand.js

```diff
@@ -21,6 +21,7 @@ import Command from '@ckeditor/ckeditor5-core/src/command';
  *
  * If the selection was anchored in a {@glink framework/guides/tutorials/implementing-a-block-widget nested editable}
  * (e.g. a caption of an image), the new selection will contain its entire content.
+ * Successive execution - selecting all if entire content is selected - expands selection to the parent editable.
```

Comment:
This is not exactly how this works. It expands selection to closest limit element and if it's content is entirely selected then tries to select closest object element: caption content -> image object -> table cell content -> table object -> root content